<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLP4500TestClass</class>
 <widget class="QMainWindow" name="DLP4500TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLP4500Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接投影仪</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FirmwareTag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_firmwareTagInfo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>xxx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="project_Button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始投影格雷码+四步相移图案</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PatSeqDispRunContinuous">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>repeat</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_PatSeqDispRunOnce">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>play once</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PatSeqPatExpTime">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ExposureTime</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PatternPeriod</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PatSeqPatPeriod">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>停止投影</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SLMode">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PatternSequence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DLP4500Test.qrc"/>
 </resources>
 <connections/>
</ui>
